src/DateRangePicker/test/DateRangePickerSpec.tsx

```diff
@@ -884,6 +884,33 @@ describe('DateRangePicker', () => {
     ).to.have.length(2);
   });
 
+  it('Should focus on the right month', () => {
+    const onEnterSpy = sinon.spy();
+
+    const App = () => {
+      const [value, setValue] = useState<[Date, Date] | null>([new Date(), new Date()]);
+      useEffect(() => {
+        setValue([new Date(2022, 10, 1), new Date(2022, 11, 1)]);
+      }, []);
+      return <DateRangePicker value={value} onChange={setValue} onEnter={onEnterSpy} />;
+    };
+    const { getByRole } = render(<App />);
+
+    fireEvent.click(getByRole('combobox'));
```

Comment:
Use `rerender` (whenever possible) to update props passed to DateRangePicker.

https://testing-library.com/docs/react-testing-library/api/#rerender